dist/elliptic.js

```diff
@@ -3935,10 +3935,7 @@ utils.intFromLE = intFromLE;
   // BN
 
   function BN (number, base, endian) {
-    // May be `new BN(bn)` ?
```

Comment:
There is no need to recompile it here, it will happen automatically on `npm version ...`. May I ask you to remove this file from this PR?